go/cmd/vtctldclient/command/vreplication/vdiff/vdiff.go

```diff
@@ -861,6 +861,7 @@ func registerVDiffCommands(root *cobra.Command) {
 	create.Flags().StringSliceVar(&createOptions.SourceCells, "source-cells", nil, "The source cell(s) to compare from; default is any available cell.")
 	create.Flags().StringSliceVar(&createOptions.TargetCells, "target-cells", nil, "The target cell(s) to compare with; default is any available cell.")
 	create.Flags().Var((*topoprotopb.TabletTypeListFlag)(&createOptions.TabletTypes), "tablet-types", "Tablet types to use on the source and target.")
+	create.Flags().BoolVar(&common.CreateOptions.TabletTypesInPreferenceOrder, "tablet-types-in-preference-order", true, "When performing source tablet selection, look for candidates in the type order as they are listed in the tablet-types flag.")
```

Comment:
This is unrelated to this PR, but in working on it I realized that I had missed adding it here: https://github.com/vitessio/vitess/pull/13976